tests/cisco_xr/show_controllers_hundredgige_all/cisco_xr_show_controllers_hundredgige_all.yml

```diff
@@ -0,0 +1,930 @@
+---
+parsed_sample:
+- admin_state: "enabled"
+  amps_alarm_high: ""
+  amps_alarm_low: ""
+  amps_warn_high: ""
+  amps_warn_low: ""
+  autonegotiation: ""
+  dom_alarms: ""
+  duplex: ""
```

Comment:
I'm not sure the template is working as intended. Looking at the raw output and the parsed data, it appears there is quite a bit of missing data: duplex, speed, mac_bia_add, and mac_ops_add are just a few examples.